downloads/swan-lake-release-notes/swan-lake-2201.11.0/RELEASE_NOTE.md

```diff
@@ -297,6 +284,109 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 11 (2201.11.0)
   }
   ```
 
+#### `data.xmldata` package
+
+- Introduced XML schema definition (XSD) Sequence and Choice support for the `data.xmldata` package.
+
+```ballerina
+  import ballerina/data.xmldata;
```

Comment:
Including comparatively longer examples may be inconsistent with the rest of the library section.